var/spack/repos/builtin/packages/py-fsspec/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyFsspec(PythonPackage):
+    """A specification for pythonic filesystems."""
+
+    homepage = "https://github.com/intake/filesystem_spec"
+    url = "https://github.com/intake/filesystem_spec/archive/0.4.4.tar.gz"
+
+    version('0.4.4', '27dfc3dab37d5c037683c7a3eaf7acd8b24ee56e4ce3edb14af54bdb43973d43')
+
+    depends_on('python@3.5:')
```

Comment:
```suggestion
    depends_on('python@3.5:', type=('build', 'run'))
```

Reply:
sorry I am not knowledgable on this:
its the same as with PR #12724: the PythonPackage-class-default seems to be blr so why to specify differently here?

Reply:
You are right that the `PythonPackage` base class defaults to `blr`. At one point I changed this to `br` but didn't have the time to test every package to make sure it didn't break anything. Basically, if it doesn't link to the Python libraries or headers (most Python packages), then it should be `br`. I'm trying to get this standard in place in the actual `package.py` files before I work on changing the base class to match.

Reply:
thanks a lot for clarifying this!